### ISSUE TITLE
FilPunks: Absence of Filecoin NFT marketplace makes it less "fun" to own an FVM NFT

### ISSUE THREAD

**Issue by lksbrssr** [FilPunks/nft-project#4]
Have you all looked into building out a basic NFT marketplace? I feel like it's a problem for several NFT projects.

**Comment by ArthurWang1255** [comment]
Yes, we need a marketplace. We thought about building a basic NFT marketplace by ourselves, but two of the engineers have quit this project so we're lack of development manpower now.
